neofetch

```diff
@@ -1346,6 +1346,10 @@ get_model() {
 
         Solaris)
             model=$(prtconf -b | awk -F':' '/banner-name/ {printf $2}')
+            virt=$(/usr/bin/uname -V)
+            if [ $? == 0 ] && [ "$virt" != "non-virtualized" ]; then
```

Comment:
This line triggers a shellcheck warning: "SC2181 (style): Check exit code directly with e.g. 'if mycmd;', not indirectly with $?."

You can fix it with the following change:

```suggestion
            if virt=$(/usr/bin/uname -V) && [[ "$virt" != "non-virtualized" ]]; then
```